ingestion/src/metadata/ingestion/source/database/redshift/queries.py

```diff
@@ -216,3 +216,58 @@
       AND n.nspname <> 'pg_catalog'
     ORDER BY "schema", "table_name";
 """
+REDSHIFT_SQL_STATEMENT_TEST = """
```

Comment:
we can break this down into simpler steps so that we don't load all the data.

Our ingredients are:
- pg_catalog.stl_query
- pg_catalog.stl_querytext
- pg_catalog.svv_table_info

We can just run something like `select * from pg_catalog.svv_table_info limit 10` for each of them

Reply:
we can have 3 steps, one for each table with a simple query